Plotter/python/plot/Stack.py

```diff
@@ -76,8 +76,8 @@ def draw(self,*args,**kwargs):
     xmax         = kwargs.get('xmax',         self.xmax       )
     ymin         = kwargs.get('ymin',         self.ymin       )
     ymax         = kwargs.get('ymax',         self.ymax       )
-    rmin         = kwargs.get('rmin',         self.rmin       ) or (0.0 if fraction else 0.45) # ratio ymin
-    rmax         = kwargs.get('rmax',         self.rmax       ) or 1.55 # ratio ymax
+    rmin         = kwargs.get('rmin',         self.rmin       ) or (0.0 if fraction else 0.8) # ratio ymin default = 0.45
```

Comment:
Can you change back these defaults for general use, please? For your specific analysis, you can set `rrange=0.2` or `rmin=0.8,rmax=1.1`.